CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Next Release
+
+- Your contribution here.
+- [#35](https://github.com/technekes/cast-ui/pull/35): Adds pagination component - [@ben-dalton](https://github.com/ben-dalton).
+
+# 0.1.8 (02/07/2019)
```

Comment:
Prefer ISO-8601 dates?

![iso-8601](https://imgs.xkcd.com/comics/iso_8601.png)

Reply:
Haha. I am partial to some of the latter options.